src/resources/parser/glb-parser.js

```diff
@@ -816,7 +871,7 @@ const createMesh = function (device, gltfMesh, accessors, bufferViews, callback,
 
                     if (target.hasOwnProperty('POSITION')) {
                         accessor = accessors[target.POSITION];
-                        options.deltaPositions = getAccessorData(accessor, bufferViews);
+                        options.deltaPositions = getAccessorData(accessor, bufferViews, true);
```

Comment:
morph positions need to be float format as well, it should use getAccessorDataFloat32

Reply:
I actually found that quantized positions worked fine under texture-based morphing, but not vertex based. I was going to submit a ticket to address this, but thinking about it a little more I suspect normalized quantized positions probably won't work, even with texture based. In that case probably converting to float32 is also necessary.

So yeah, unfortunately we may have to dequantize everything upfront for now. Implementing quantized/normlized support in all mprphing render paths will be a big job (and maintenance nightmare).